### PR TITLE
feat(docs): extend LE instructions

### PR DESCRIPTION
**Pull request by barrydegraaff**

### CODE REVIEW

**Comment by barrydegraaff** [comment]
@nils-wisiol moved from https://github.com/desec-io/desec-stack/pull/457

File: docs/dyndns/lets-encrypt.rst [hunk 1]
```diff
@@ -42,6 +48,13 @@ steps.
 
      certbot --manual --manual-auth-hook ./hook.sh --manual-cleanup-hook ./hook.sh \
          --preferred-challenges dns -d "YOURDOMAINNAME.dedyn.io" certonly
+         
+   You can also use certbot to get wildcart certificates like so:
```
Comment (peterthomassen) [inline docs/dyndns/lets-encrypt.rst:52]: wildcard

**Review by peterthomassen — CHANGES_REQUESTED** [review]
Great, thanks for the PR! It's really appreciated <3

I have one small typo correction, otherwise good to go. Once you fixed it, you can refresh this PR (without creating a merge commit) by commit the change with `git commit --amend`, and then overriding your branch with the new commit using `git push -f`.

**Review by peterthomassen — APPROVED** [review]
lgtm!

**Comment by nils-wisiol** [comment]
thanks again, @barrydegraaff :rocket:

**Pull request merged**